patch-tracing-channel.js

```diff
@@ -98,15 +98,15 @@ module.exports = function (unpatched) {
         context.error = err;
         error.publish(context);
         asyncStart.publish(context);
-        // TODO: Is there a way to have asyncEnd _after_ the continuation?
```

Comment:
Removing the two TODO comments in this file since it's from upstream code and not a TODO of this project. That way we can grep.